bin/sgqlc-codegen

```diff
@@ -274,6 +275,13 @@ class %(name)s(sgqlc.types.Enum):
         else:
             return repr(name)
 
+    def get_py_fields_and_siblings(self, own_fields):
+        graphql_fields = tuple(field['name'] for field in own_fields)
```

Comment:
nit it may be better to define `graphql_fields` as a set from the start. It would protect `py_fields` against duplicated values in `own_fields` if that were to happen.

Reply:
it's not likely and I'm afraid order would be changed

Reply:
All right then